From 89382454060c4fca96e033b7a3afb84497a77a7d Mon Sep 17 00:00:00 2001
Message-Id: <89382454060c4fca96e033b7a3afb84497a77a7d.1493300240.git.devel@agrez.net>
From: Vaughan Agrez <devel@agrez.net>
Date: Thu, 27 Apr 2017 21:02:30 +0800
Subject: [PATCH] Update panel.conf

---
 panel/resources/panel.conf | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/panel/resources/panel.conf b/panel/resources/panel.conf
old mode 100644
new mode 100755
index e833229..7751903
--- a/panel/resources/panel.conf
+++ b/panel/resources/panel.conf
@@ -1,7 +1,9 @@
 panels=panel1
 
 [panel1]
-plugins=mainmenu,desktopswitch,quicklaunch,taskbar,tray,statusnotifier,mount,volume,clock,showdesktop
+iconSize=26
+panelSize=38
+plugins=mainmenu, quicklaunch, desktopswitch, taskbar, tray, statusnotifier, volume, clock
 position=Bottom
 desktop=0
 
@@ -13,6 +15,10 @@ type=desktopswitch
 
 [quicklaunch]
 type=quicklaunch
+apps\1\desktop=/usr/share/applications/qterminal.desktop
+apps\2\desktop=/usr/share/applications/pcmanfm-qt.desktop
+apps\3\desktop=/usr/share/applications/firefox.desktop
+apps\size=3
 alignment=Left
 
 [taskbar]
@@ -28,6 +34,7 @@ type=mount
 type=clock
 
 [volume]
+audioEngine=Alsa
 device=0
 type=volume
 
-- 
2.9.3

