diff -U 3 -H -d -r -N -- lxqt-panel-0.11.1.orig/panel/ilxqtpanel.h lxqt-panel-0.11.1/panel/ilxqtpanel.h
--- lxqt-panel-0.11.1.orig/panel/ilxqtpanel.h	2017-01-02 06:27:14.000000000 +0800
+++ lxqt-panel-0.11.1/panel/ilxqtpanel.h	2018-04-02 22:29:30.771548472 +0800
@@ -78,7 +78,7 @@
      * @return The QRect where the panel is located in global screen
      * coordinates.
      */
-    virtual QRect globalGometry() const = 0;
+    virtual QRect globalGeometry() const = 0;
 
     /**
      * @brief Helper method for calculating the global screen position of a
diff -U 3 -H -d -r -N -- lxqt-panel-0.11.1.orig/panel/lxqtpanel.cpp lxqt-panel-0.11.1/panel/lxqtpanel.cpp
--- lxqt-panel-0.11.1.orig/panel/lxqtpanel.cpp	2017-01-02 06:27:14.000000000 +0800
+++ lxqt-panel-0.11.1/panel/lxqtpanel.cpp	2018-04-02 22:29:36.491562091 +0800
@@ -950,9 +950,10 @@
 /************************************************
 
  ************************************************/
-QRect LXQtPanel::globalGometry() const
+QRect LXQtPanel::globalGeometry() const
 {
-    return QRect(mapToGlobal(QPoint(0, 0)), this->size());
+    // panel is the the top-most widget/window, no calculation needed
+    return geometry();
 }
 
 
@@ -1116,19 +1117,19 @@
     switch (position())
     {
     case ILXQtPanel::PositionTop:
-        y = globalGometry().bottom();
+        y = globalGeometry().bottom();
         break;
 
     case ILXQtPanel::PositionBottom:
-        y = globalGometry().top() - windowSize.height();
+        y = globalGeometry().top() - windowSize.height();
         break;
 
     case ILXQtPanel::PositionLeft:
-        x = globalGometry().right();
+        x = globalGeometry().right();
         break;
 
     case ILXQtPanel::PositionRight:
-        x = globalGometry().left() - windowSize.width();
+        x = globalGeometry().left() - windowSize.width();
         break;
     }
 
@@ -1163,7 +1164,8 @@
     if (nullptr == panel_plugin)
         return QRect();
 
-    return calculatePopupWindowPos(panel_plugin->mapToGlobal(QPoint(0, 0)), windowSize);
+    // Note: assuming there are not contentMargins around the "BackgroundWidget" (LXQtPanelWidget)
+    return calculatePopupWindowPos(globalGeometry().topLeft() + panel_plugin->geometry().topLeft(), windowSize);
 }
 
 
diff -U 3 -H -d -r -N -- lxqt-panel-0.11.1.orig/panel/lxqtpanel.h lxqt-panel-0.11.1/panel/lxqtpanel.h
--- lxqt-panel-0.11.1.orig/panel/lxqtpanel.h	2017-01-02 06:27:14.000000000 +0800
+++ lxqt-panel-0.11.1/panel/lxqtpanel.h	2018-04-02 22:29:30.772548474 +0800
@@ -142,7 +142,7 @@
 
     // ILXQtPanel overrides ........
     ILXQtPanel::Position position() const override { return mPosition; }
-    QRect globalGometry() const override;
+    QRect globalGeometry() const override;
     QRect calculatePopupWindowPos(QPoint const & absolutePos, QSize const & windowSize) const override;
     QRect calculatePopupWindowPos(const ILXQtPanelPlugin *plugin, const QSize &windowSize) const override;
     void willShowWindow(QWidget * w) override;
diff -U 3 -H -d -r -N -- lxqt-panel-0.11.1.orig/plugin-clock/lxqtclock.cpp lxqt-panel-0.11.1/plugin-clock/lxqtclock.cpp
--- lxqt-panel-0.11.1.orig/plugin-clock/lxqtclock.cpp	2017-01-02 06:27:14.000000000 +0800
+++ lxqt-panel-0.11.1/plugin-clock/lxqtclock.cpp	2018-04-02 22:29:30.772548474 +0800
@@ -260,7 +260,7 @@
         size.setWidth(metrics.boundingRect(QString{mCurrentCharCount, 'A'}).width());
     } else if (!panel()->isHorizontal())
     {
-        size.setWidth(panel()->globalGometry().width());
+        size.setWidth(panel()->globalGeometry().width());
     }
 
     mTimeLabel->setFixedWidth(size.width());
