--- lxqt-common-0.13.0/menu/lxqt-applications.menu.orig
+++ lxqt-common-0.13.0/menu/lxqt-applications.menu
@@ -13,6 +13,26 @@
 	<!-- Read in overrides and child menus from applications-merged/ -->
 	<DefaultMergeDirs/>
 
+	<!-- RedHat special submenu -->
+	<Menu>
+		<Name>System Settings</Name>
+		<Directory>SystemConfig.directory</Directory>
+		<Include>
+			<And>
+				<Category>System</Category>
+				<Category>Settings</Category>
+				<Not><Category>X-Red-Hat-ServerConfig</Category></Not>
+			</And>
+		</Include>
+		<Menu>
+			<Name>Server</Name>
+			<Directory>ServerConfig.directory</Directory>
+			<Include>
+				<Category>X-Red-Hat-ServerConfig</Category>
+			</Include>
+		</Menu>
+	</Menu>
+
 	<!-- Accessories submenu -->
 	<Menu>
 		<Name>Accessories</Name>
@@ -138,6 +158,7 @@
 				<Category>System</Category>
 				<Not><Category>Settings</Category></Not>
 				<Not><Category>PackageManager</Category></Not>
+				<Not><Category>X-Red-Hat-ServerConfig</Category></Not>
 				<Not><Category>X-Leave</Category></Not>
 				<Not><Category>Screensaver</Category></Not>
 				<Not><Filename>lxqt-about.desktop</Filename></Not>
