diff -up lxqt-panel-0.10.0/CMakeLists.txt.than lxqt-panel-0.10.0/CMakeLists.txt
diff -up lxqt-panel-0.10.0/plugin-sysstat/CMakeLists.txt.than lxqt-panel-0.10.0/plugin-sysstat/CMakeLists.txt
--- lxqt-panel-0.10.0/plugin-sysstat/CMakeLists.txt.than	2016-05-24 17:39:16.444674669 +0200
+++ lxqt-panel-0.10.0/plugin-sysstat/CMakeLists.txt	2016-05-24 17:40:18.456847629 +0200
@@ -1,6 +1,7 @@
 set(PLUGIN "sysstat")
 
 find_package(SysStat-Qt5 REQUIRED)
+include_directories("${SYSSTAT_INCLUDE_DIR}")
 
 set(HEADERS
     lxqtsysstat.h
