diff -U 3 -H -d -r -N -- lxqt-panel-0.11.1.orig/plugin-volume/lxqtvolumeconfiguration.h lxqt-panel-0.11.1/plugin-volume/lxqtvolumeconfiguration.h
--- lxqt-panel-0.11.1.orig/plugin-volume/lxqtvolumeconfiguration.h	2017-01-02 06:27:14.000000000 +0800
+++ lxqt-panel-0.11.1/plugin-volume/lxqtvolumeconfiguration.h	2018-03-25 19:56:10.134845526 +0800
@@ -46,7 +46,7 @@
 #define SETTINGS_DEFAULT_DEVICE                 0
 #define SETTINGS_DEFAULT_STEP                   3
 #ifdef USE_PULSEAUDIO
-    #define SETTINGS_DEFAULT_MIXER_COMMAND      "pavucontrol"
+    #define SETTINGS_DEFAULT_MIXER_COMMAND      "pavucontrol-qt"
     #define SETTINGS_DEFAULT_AUDIO_ENGINE       "PulseAudio"
 #elif defined(USE_ALSA)
     #define SETTINGS_DEFAULT_MIXER_COMMAND      "qasmixer"
