--- a/panel/resources/panel.conf.old	2018-05-22 03:24:34.000000000 +0800
+++ b/panel/resources/panel.conf	2018-11-11 19:38:49.454407778 +0800
@@ -1,11 +1,15 @@
 panels=panel1
 
 [panel1]
-plugins=mainmenu,desktopswitch,quicklaunch,taskbar,tray,statusnotifier,mount,volume,worldclock,showdesktop
+iconSize=26
+panelSize=38
+plugins=mainmenu, quicklaunch, desktopswitch, taskbar, tray, statusnotifier, volume, worldclock
 position=Bottom
 desktop=0
 
 [mainmenu]
+showText=true
+text=Menu
 type=mainmenu
 
 [desktopswitch]
@@ -13,6 +17,10 @@
 
 [quicklaunch]
 type=quicklaunch
+apps\1\desktop=/usr/share/applications/qterminal.desktop
+apps\2\desktop=/usr/share/applications/pcmanfm-qt.desktop
+apps\3\desktop=/usr/share/applications/chromium-browser.desktop
+apps\size=3
 alignment=Left
 
 [taskbar]
@@ -28,6 +36,7 @@
 type=worldclock
 
 [volume]
+audioEngine=Alsa
 device=0
 type=volume
 
